projects/atft/karma.conf.js

```diff
@@ -0,0 +1,32 @@
+// Karma configuration file, see link for more information
```

Comment:
Let's see if that works with the e2e tests. I had to adjust the previous `karma.conf.js` to be able to run the e2e tests, also on Travis.